core/tests/test_utils.py

```diff
@@ -1524,6 +1525,10 @@ def save_new_exp_with_states_schema_v0(self, exp_id, user_id, title):
         )
         exp_summary_model.put()
 
+        # Create an ExplorationIssues model to match the behavior of creating
+        # new explorations.
+        stats_services.create_exp_issues_for_new_exploration(exp_id, 1)
```

Comment:
When does this happen in normal production scenarios? Just making sure that this line should be here to match standard prod behavior.

Reply:
It happens on this line: https://github.com/oppia/oppia/blob/fc48b2c1cd295a8790247aad3ec52896f5741db0/core/domain/exp_services.py#L617-L619

Reply:
Thanks!